docs/developers/configuring-qps-burst-thread.md

```diff
@@ -0,0 +1,42 @@
+# Configure ThreadsPerController, QPS and Burst
```

Comment:
I don't think the `developers` subdirectory is the right place for this doc.  `developers` is for documentation that people coding Tekton Pipelines might need.

Suggest putting this file at `docs/performance-configuration.md` or something similar. What do you think?

Reply:
@sbwsg Thanks for your suggestions! 

Actually, I agree with you. But I saw this doc `enabling-ha.md` also under `developers` folder. I think these two files have the same target. So I also put it in `developers` subdirectory. Anyway, I can change it to `docs/performance-configuration.md`. Thanks again!

Reply:
Done! Pls take a look again, thanks!

Reply:
Ah, you're right.  I think `enabling-ha.md` should probably also be in the docs section.  I will make a PR, thank you!